third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -416,6 +417,92 @@ func TestAccBigQueryDataTable_sheet(t *testing.T) {
 	})
 }
 
+func TestUnitBigQueryDataTable_jsonEquivelency(t *testing.T) {
+	t.Parallel()
+	if len(testUnitBigQueryDataTableJSONEquivelencyResults) != len(testUnitBigQueryDataTableJSONEquivelencyData) {
+		t.Error("these arrays aren't equal length! whatchu doing son?!?")
+	}
+
+	for i, testcase := range testUnitBigQueryDataTableJSONEquivelencyData {
+		var a, b interface{}
+		json.Unmarshal([]byte(testcase[0]), &a)
+		json.Unmarshal([]byte(testcase[1]), &b)
+		eq, err := jsonCompareWithMapKeyOverride(a, b, bigQueryTableMapKeyOverride)
+		if err != nil {
+			t.Errorf("ahhhh an error I did not expect this! especially not on testscase %v", i)
+		}
+		if eq != testUnitBigQueryDataTableJSONEquivelencyResults[i] {
+			t.Errorf("expected equivelency result of %v but got %v for testcase number %v", testUnitBigQueryDataTableJSONEquivelencyResults[i], eq, i)
+		}
+	}
+}
+
+var testUnitBigQueryDataTableJSONEquivelencyResults = []bool{
```

Comment:
A common way of doing this is 
[]struct{
  jsonA string
  jsonB string
  equivalent bool
}{
{
  "foo", "bar", false
},
...
}


Which I find a lot easier to read.

Reply:
You can use three backticks and put code directly inline 
```
hello, world. #$
```

Reply:
done